src/Diagnostics.ModelsAndUtils/Models/Search/SearchModels.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Diagnostics.ModelsAndUtils.Attributes;
+using Diagnostics.ModelsAndUtils.ScriptUtilities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models
```

Comment:
Add documentation for properties if possible.

Reply:
Done.